src/_macro.ts

```
@@ -2,8 +2,8 @@
  * Prefetch claude data for the current user.
  */
 
-import { LITELLM_PRICING_URL } from './consts.internal.ts';
-import { type ModelPricing, modelPricingSchema } from './types.internal.ts';
+import { LITELLM_PRICING_URL } from './_consts.ts';
+import { type ModelPricing, modelPricingSchema } from './_types.ts';
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)